### PR TITLE
chore(deps): general maintenance

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[][demo] |
:---:|

## 🧰 Changes

This PR removes some packages, updates some, and updates what versions of node CI is run against. node v10 has been EOL for about 2 months.

The following packages have been removed as their newer replacement is already installed:

- remove babel-core
- remove babel-eslint
- remove babel-polyfill

## 🧬 QA & Testing

- [Working in this PR app][demo].

[demo]: https://markdown-pr-186.herokuapp.com

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [12.x, 14.x, 16.x]
```
Comment (erunion) [inline .github/workflows/ci.yml:10]: Make sure to update the release workflow too, it's still on Node 10: https://github.com/readmeio/markdown/blob/next/.github/workflows/release.yml#L15

File: package.json [hunk 2]
```diff
@@ -73,11 +75,9 @@
     "@readme/variable": "^13.4.12",
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/git": "^9.0.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.1.0",
     "babel-jest": "^27.0.5",
     "babel-loader": "^8.2.2",
-    "babel-polyfill": "^6.26.0",
+    "chokidar": "^3.5.2",
```
Comment (erunion) [inline package.json:80]: Where are you using Chokidar?

**Review by erunion — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kellyjosephprice) [reply .github/workflows/ci.yml:10]: Oh thanks!

File: package.json (hunk 2 above)
Comment (kellyjosephprice) [reply package.json]: It's a dep of webpack-dev-server. I **was** getting a `npm i` error, but it seems fine now.

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.28.0-next.1</b>](/readmeio/markdown/releases/tag/v6.28.0-next.1)
<br>

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.28.0</b>](/readmeio/markdown/releases/tag/v6.28.0)
<br>

**Pull request merged**